src/config/Config.zig

```diff
@@ -1336,6 +1336,26 @@ keybind: Keybinds = .{},
 ///   * `end` - Insert the new tab at the end of the tab list.
 @"window-new-tab-position": WindowNewTabPosition = .current,
 
+/// Whether to show the tab bar.
+/// Currently only supported on Linux (GTK).
```

Comment:
A very small nit: this text is usually shown in a separate line at the very bottom of the config docs idiomatically.